torch/serialization.py

```diff
@@ -1886,15 +1888,79 @@ def _load(
             UserWarning,
         )
 
+    from torch.utils.serialization import config
+
+    calculate_storage_offsets = config.load.calculate_storage_offsets
+    run_debug_asserts = os.environ.get("TORCH_SERIALIZATION_DEBUG", "0") == "1"
+    current_offset = None
+    data_descriptor_size64 = 24
+    data_descriptor_size32 = 16
+    offsets: Dict[str, int] = dict()
+
+    def _get_offset(key, name, numel):
+        """
+        Return the offset of the storage associated with key with record name `name` and size numel.
```

Comment:
nit: add a warning here and in C++ mentioning that the two must be kept in sync !